config/database.yml

```diff
@@ -13,7 +13,7 @@ test:
   database: open_food_network_test<%= ENV['TEST_ENV_NUMBER'] %>
   pool: 5
   host: localhost
-  username: ofn
+  username: ofn_user
```

Comment:
Why is this changing? This will break my local dev environment...

Reply:
Damn. Didn't mean to commit that. Will remove it.